nano/node/node.cpp

```diff
@@ -658,6 +658,7 @@ void nano::node::start ()
 	}
 	websocket.start ();
 	telemetry.start ();
+	local_block_broadcaster.start ();
```

Comment:
I think it should be stopped first rather than last because it depends on other components.

Reply:
A stopped component should still be perfectly valid, but it can be stopped first, it shouldn't matter

Reply:
I see your point. It is not a destructor.